tinygrad/codegen/uopgraph.py

```diff
@@ -477,7 +497,7 @@ def loop_collapse(loop_start, loop_end, compval, idx, mval, multconst, rng):
   (UOp.load(UOp.var("buf"), UOp.var("idx"), UOp.const(dtypes.bool, True), UOp.cvar("var")), lambda buf,idx,var: UOp.load(buf, idx, dtype=var.dtype)),
   (UOp.load(UOp.var("buf"), UOp.var("idx"), UOp.const(dtypes.bool, True), UOp.cvar("var"), UOp.var("barrier")),
    lambda buf,idx,var,barrier: UOp.load(buf, idx, barrier, dtype=var.dtype)),
-  (UOp.load(UOp.var(), UOp.var(), UOp.const(dtypes.bool, False), UOp.cvar("var")), lambda var: var),
+  (UOp.load(UOp.var(), UOp.var(), UOp.const(dtypes.bool, False), UOp.var("var")), lambda var: var),
   (UOp.load(UOp.var(), UOp.var(), UOp.const(dtypes.bool, False), UOp.cvar("var"), UOp.var()), lambda var: var),
```

Comment:
wanna also change this to var?

Reply:
Thanks, I was thinking about this but unsure if I should since I'm unfamiliar with these UPats. Made the suggested change. If I may ask, why are you referring to the uop as `invalid`? I don't see any references to `invalid` in the code.

Reply:
"invalid" refers to a CONST 0 for a padded LOAD:
https://github.com/tinygrad/tinygrad/blob/master/tinygrad/codegen/lowerer.py#L119-L124

before, it was CONST(0, dtypes.float.vec(4))
Now it's VECTORIZE(CONST(0), dtypes.float.vec(4)) so your UOp for "var" changed.